src/js/shield_defs.js

```diff
@@ -3385,6 +3385,10 @@ export function loadShields(shieldImages) {
       bottom: 7,
     },
   };
+  shields["IN:NE"] = {
+    ...shields["IN:NH"],
+    numberingSystem: "roman",
+  };
```

Comment:
I considered adding an “NE” banner up top, similar to the “TWP” disambiguation for some Ohio township routes, because this shield looks so similar to the NH shield at a glance. However, the Roman numerals are probably sufficient.

Reply:
I added an “NE” banner:

[<img src="https://user-images.githubusercontent.com/1231218/211967116-5e52f378-af8b-4e22-b7a7-89069b03f28c.png" width="400" alt="Ghaziabad">](https://zelonewolf.github.io/openstreetmap-americana/#map=11/28.6911/77.535)